docs/source/docs/quick-start/camera-calibration.md

```diff
@@ -0,0 +1,29 @@
+# Camera Calibration
+
+:::{important}
+In order to detect AprilTags and use 3D mode, your camera must be calibrated at the desired resolution! Inaccurate calibration will lead to poor performance.
+:::
+
+If you’re not using cameras in 3D mode, calibration is optional, but it can still offer benefits. Calibrating cameras helps refine the pitch and yaw values, leading to more accurate positional data in every mode. {ref}`For a more in-depth view<docs/calibration/calibration:Calibrating Your Camera>`.
+
+## Print the Calibration Target
+
+- Downloaded from our [demo site](https://demo.photonvision.org/#/cameras), or directly from your coprocessors cameras tab.
+- Use the Charuco calibration board:
+  - Layout: 8x8 grid of 1-inch squares.
```

Comment:
this is confusing and doesn't match the UI well

Reply:
there are other options that should either be mentioned to users that these are things that you can do or something?

Reply:
We are STRONGLY recommending charuco since there are some slight bugs with the standard grid calibration.